src/Aspire.CommunityToolkit.Hosting.Azure.DataApiBuilder/DataApiBuilderContainerImageTags.cs

```diff
@@ -0,0 +1,8 @@
+namespace Aspire.CommunityToolkit.Hosting.Azure.DataApiBuilder;
+
+internal static class DataApiBuilderContainerImageTags
+{
+    public const string Registry = "mcr.microsoft.com";
+    public const string Image = "azure-databases/data-api-builder";
+    public const string Tag = "1.2.11";
```

Comment:
How is this kept up to date?

Reply:
We'll follow the same approach that the Aspire team does. When a new image tag is published we'll create an issue, someone on the team (or from the community) will create a PR and our tests will run against it and a new release will be prepared once it's passed review.

But if someone needs to use the new tagged image prior to it being shipped officially, the `WithImageTag` method on a `ContainerResource` in Aspire can be used to override the image tag specified by the integration (this can also be used to downgrade if someone needs to).